orbyter-ml-dev/1.3/requirements.txt

```diff
@@ -0,0 +1,32 @@
+black==19.3b0
+boto3==1.9.127
+Click==7.0
+coloredlogs==10.0
+cufflinks==0.14.6
+dask==1.1.5
+distributed==1.26.1
+fire==0.1.3
+flake8==3.7.7
+ipdb==0.12
+isort==4.3.16
+jsonlines==1.2.0
+jupyter==1.0.0
+jupyter-contrib-nbextensions==0.5.1
+lightgbm==2.2.3
+matplotlib==3.0.3
+mlflow==0.9.0
+more-itertools==7.0.0
+numpy==1.16.2
+pandas==0.24.2
+plotly==3.7.1
+pluggy==0.9.0
+psycopg2==2.8.1
```

Comment:
did you get complaints about using psycopg2-binary instead of this?

Reply:
Yes. I was able to fix the issue in the Dockerfile: `apt-get install -y libpq-dev python3-dev &&\`